daffodil-test/src/test/scala-new/edu/illinois/ncsa/daffodil/section07/external_variables/TestExternalVariablesNew.scala

```diff
@@ -53,4 +53,12 @@ class TestExternalVariablesNew {
     runner.runOneTest("read_config_from_file")
   }
 
+  //
+  // Use to reproduce CLI test test_2360_CLI_Parsing_SimpleParse_stdOut_extVars2
+  // TODO: When Daffodil-1846 is fixed this test could be enabled.
+  // @Test
+  def test_testNoRootUnnecessaryBinding(): Unit = {
+    runner.trace.runOneTest("testNoRootUnnecessaryBinding")
+  }
+
```

Comment:
Rather than commenting out @Test, can this be put in scala-debug? Makes it easier to run the test when someone goes to fix it. Also, makes it easier to just run all debug tests and see if tests started working due and we just forgot to move a test to scala.